docs/dev-machine-setup.md

```diff
@@ -11,11 +11,53 @@ outlined in the [Getting Started](getting-started.md) instructions).
 
 ### Local Bare-Metal Machine
 
-`[TODO]`
+
 
 ### Local Virtual Machine
 
-`[TODO]`
+Firecracker can also be installed in a Proxmox VM which uses KVM as Hypervisor. The nested option on the host where Proxmox is running must be enabled. In the following example nested virtualization has not been enabled yet ("N" No) so to check nested virtualization in your Proxmox host excecute the following command:
+
+   ```
+   cat /sys/module/kvm_intel/parameters/nested
+   N
+   ```
+
+To enable nested virtualization follow these steps:
+
+   ```
```

Comment:
nit: It looks better with one block of code rather than split each command in one block:
```
   echo "options kvm-intel nested=Y" > /etc/modprobe.d/kvm-intel.conf
   modprobe -r kvm_intel
   modprobe kvm_intel
   reboot
```